src/templates/pycontw-2019/base.html

```diff
@@ -15,7 +15,7 @@
 <title>{% block title %}{% endblock title %} | PyCon Taiwan 2019</title>
 {% endblock titletag %}
 
-<link rel="shortcut icon" href="{% static 'pycontw-2019/assets/favicon.ico' %}">
+<link rel="shortcut icon" href="{% static 'pycontw-2019/assets/favicon.png' %}">
```

Comment:
Per [best practice](https://github.com/audreyr/favicon-cheat-sheet#the-images), there should be at least one .ico favicon. You can add as many .png favicons as you want, but please the .ico one.